keyboards/dumbpad/v1x_dualencoder/readme.md

```diff
@@ -0,0 +1,87 @@
+# dumbpad v1.x dual-encoder
+
+![dumbpad](v1x_dualencoder.png)
```

Comment:
The photo here should be uploaded to some place like Imgur and linked here, and then `keyboards/dumbpad/v1x_dualencoder/v1x_dualencoder.png` should be deleted.

From the [Docs](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=imagehardware-files "QMK Keyboard Guidelines"):

> In an effort to keep the repo size down we’re no longer accepting binary files of any format, with few exceptions. Hosting them elsewhere (such as https://imgur.com) and linking them in the readme.md is preferred.

Reply:
Done!